contrib/export/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/tendermint/tendermint/types"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"v0.36": v036.Migrate,
+	}
+	source        string
+	target        string
+	importGenesis string
+)
+
+func init() {
+	log.SetPrefix("")
+	log.SetFlags(0)
+
+	// this flag will be used once we support more than one previous version
+	flag.StringVar(&source, "source", "", "The SDK version that exported the genesis")
+
+	flag.Usage = func() {
+		_, _ = fmt.Fprintf(flag.CommandLine.Output(),
+			`Usage: %s [v0.36] [genesis.json] [-source v0.34]
+Migrate the source genesis into the target version and export it as standard output
+`, filepath.Base(os.Args[0]))
+		flag.PrintDefaults()
+	}
+}
+
+func main() {
+	flag.Parse()
```

Comment:
Why not use cobra? Are we against that?

Reply:
Blame me please, I advised against it :)
For two positional arguments, one flag and zero sub commands it seemed an overkill to use cobra

Reply:
I had the same feeling, mostly because it's running separately from other CLI commands.
Conflicting between:
- simplifiying (simple flag)
- standardizing (cobra everywhere)

Reply:
Yeah but you get all these extra flags when you do it this way (see `go run main.go --help`).

Reply:
True, ok let's switch to Cobra!